### DISCUSSION TITLE
mark target as updated/not-to-be-rerun?

### DISCUSSION THREAD

**Discussion by bbolker** [ropensci/targets#611] [Help] [answered]
Suppose I make some changes in an upstream function that I know are organizational/won't change a downstream target in any important way.  If I were using `make` (which relies on timestamps, not hashes, and is file-based) I could simply `touch` the downstream file. 

As far as I can tell the best way to do this in `targets` is to add `cue = tar_cue(depend = FALSE)` to the target definition temporarily, and then remove it when I make a substantive change upstream.

Is there a better/more idiomatic way that I've missed?

**Comment by wlandau** [comment, marked as answer]
I think `cue = tar_cue(mode = "never")` will do what you want.

**Reply by bbolker** [reply]
OK.  It would be nice to be able to do this once/transiently , but it's good to have a solution in any case.

**Reply by wlandau** [reply]
You can set this as the default behavior with `tar_option_set(cue = tar_cue(mode = "never"))`. Then you can overwrite this cue on a target-by-target basis.
